src/sentry/processing/backpressure/arroyo.py

```diff
@@ -53,7 +49,6 @@ def create_backpressure_step(
 
     def ensure_healthy_queue(message: Message[TPayload]) -> TPayload:
         if not health_checker.is_healthy():
-            time.sleep(SLEEP_MS / 1000)
```

Comment:
We ended up backporting this sleep to Arroyo so clients don't need to do it themselves